libs/k8s-client/composer.json

```
@@ -21,9 +21,9 @@
     },
     "require": {
         "php": "^8.1",
-        "guzzlehttp/psr7": "<2.0",
         "keboola/retry": "^0.5.1",
-        "kubernetes/php-client": "^1.20"
+        "kubernetes/php-client": "1.25.16",
+        "kubernetes/php-runtime": "dev-dev-1.24.0#c3134c7 as 1.0.9"
```

```suggestion
        "kubernetes/php-runtime": "1.0.9"
```
Uz jsem mergnul a relasnul svuj PR